mpas_analysis/ocean/time_series_antarctic_melt.py

```diff
@@ -1,18 +1,25 @@
 import os
 import xarray
 
+# SFP: the following are only needed for the stop-gap local plotting routine used here
+import matplotlib.pyplot as plt
+from matplotlib.collections import PatchCollection
+from shapely.geometry.polygon import Polygon
```

Comment:
@stephenprice, so this was a good guess but it's not the `Polygon` we need. Should be:
```python
from matplotlib.patches import Polygon
```
I think that's what's causing your error.

Reply:
If you take a look at the example I emailed you from POPSICLES, you can see which modules I import there.

Reply:
Ok, that works now. Thanks! I will try to tidy up the csv file so that we can re-test applying to all of the ice shelves we are modeling.

![melt_flux_totten](https://user-images.githubusercontent.com/4983771/30231526-9b10f024-94a8-11e7-942a-f6e0ca54f0bf.png)